sqlserver/metadata.csv

```diff
@@ -69,6 +69,10 @@ sqlserver.index.user_updates,count,,update,,"Number of updates by user queries.
 sqlserver.latches.latch_wait_time,gauge,,millisecond,,Average latch wait time (in milliseconds) for latch requests that had to wait. (Perf. Counter: `Locks - Average Latch Wait Time (ms)`),0,sql_server,avg latch wait time,
 sqlserver.latches.latch_waits,gauge,,request,second,Number of latch requests that could not be granted immediately. (Perf. Counter: `Locks - Latch Waits/sec`),-1,sql_server,latch waits,
 sqlserver.locks.deadlocks,gauge,,request,second,Number of lock requests per second that resulted in a deadlock. (Perf. Counter: `Locks - Number of Deadlocks/sec`),0,sql_server,deadlocks,
+sqlserver.log_shipping_primary.time_since_backup,gauge,,second,,Seconds since the last transaction log backup operation on the primary server. (Tags: `primary_db`, `primary_server`, `primary_id`),0,sql_server,seconds since backup,
```

Comment:
You have extra commas in these lines. Enclose any string containing commas in quotes so that the CSV parser recognizes it as a single string instead of breaking it up.